tests/Unit/Entity/EventTest.php

```diff
@@ -6,15 +6,15 @@
 
 use App\Entity\Event;
 use App\Entity\EventTranslation;
+use DateTimeImmutable;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
 use Sulu\Bundle\MediaBundle\Entity\MediaInterface;
 
 class EventTest extends TestCase
 {
-    /**
-     * @var Event
-     */
-    private $event;
+    use ProphecyTrait;
+    private Event $event;
```

Comment:
php-cs-fixer should normally add here an empty line?

Reply:
looks like it does not in this case - will add it manually